lambdas/ssm-parameters/tests/ssm-parameters-handler.test.ts

```diff
@@ -0,0 +1,104 @@
+import { SsmParametersHandler } from "../src/ssm-parameters-handler";
+import { Context } from "aws-lambda";
+import { jest } from "@jest/globals";
+import { SSMProvider } from "@aws-lambda-powertools/parameters/ssm";
+
+describe("ssm-parameters-handler", () => {
+  const ssmParametersHandler = new SsmParametersHandler();
+  const ssmProvider = jest.mocked(SSMProvider).prototype;
```

Comment:
jest.mocked is preferrable with modules, SSMProvider is a class
Maybe
```
  const ssmProvider = new SSMProvider();
  jest.spyOn(ssmProvider, "getParametersByName").mockResolvedValue({}); 

```

Reply:
I think this is fine, we can use `jest.mocked()` with classes. We do this in another test:

https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/cb85f8f680551e4ad981d0707526881d8fda0097/lambdas/jwt-signer/tests/jwt-signer-handler.test.ts#L16